### ISSUE TITLE
support HEVC ？

### ISSUE THREAD

**Issue by xuyang12** [bitmovin/bitmovin-api-sdk-javascript#1]

**Comment by seburgi** [comment]
Yes, we support HEVC / H265, you can find it in encoding -> configurations -> video -> h265

[H265Api](https://github.com/bitmovin/bitmovin-api-sdk-javascript/blob/master/src/encoding/configurations/video/h265/H265Api.ts)

For more infos on this topic and other codecs pls see this blog post, we support all codecs mentioned there: https://bitmovin.com/multi-codec-world-2020/
